.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: circleci/node:8.10
+    working_directory: ~/singnet/token-contracts
+    steps:
+    - checkout
+    - run:
+        name: Build token-contracts
+        command: |
+          npm install
+          npm run-script compile
```

Comment:
Same as https://github.com/singnet/platform-contracts/pull/35/files#r220260344

Reply:
I have removed the project compilation from the CircleCI script.